.github/workflows/deploy.yml

```diff
@@ -134,12 +131,14 @@ jobs:
           setwd("${{ matrix.directory }}")
           lockfile <- renv::lockfile_read()
           pkg_name_structure <- ifelse("${{ matrix.channel }}" == "stable", "%s/%s@*release", "%s/%s")
+          unreleased_packages <- c(
+            "osprey", "hermes", "goshawk", "teal.modules.general", "teal.modules.clinical",
+            "teal.osprey", "teal.modules.hermes", "teal.goshawk"
+          )
           for (package in lockfile$Packages) {
               if (package$Source == "GitHub") {
-                  # TODO: After teal.slice is released we have to remove this change.
-                  # This workaround is because update to {bslib} breaks current release of teal.slice
-                  if (package$Package == "teal.slice" && "${{ matrix.channel }}" == "stable") {
-                    renv::record("insightsengineering/teal.slice@103491c67e26a9a85d636bd105a806904a0111ec")
+                  if (package$Package %in% unreleased_packages) {
+                    renv::record(sprintf("%s/%s", package$RemoteUsername, package$Package))
```

Comment:
Making sure that the `unreleased_packages` are taken from the `main` branch instead of the latest release version.